runtime/plugin/spellfile.lua

```diff
@@ -0,0 +1,10 @@
+require('spellfile').setup()
+
+vim.api.nvim_create_autocmd('SpellFileMissing', {
+  callback = function()
+    local lang = vim.api.nvim_exec2('echo &spelllang', { output = true })
```

Comment:
`vim.bo.spelllang` (ideally using `args.bufnr` from the callback argument)

Reply:
Done in e1add1394